server/routes/taskList.js

```diff
@@ -53,7 +54,12 @@ module.exports = function({logger, prisonerService, licenceService, authenticati
 
         const prisonerImage = await prisonerService.getPrisonerImage(req.params.imageId, req.user.token);
 
-        res.send({...prisonerImage});
+        if (!prisonerImage) {
+            const placeHolder = path.join(__dirname, '../../assets/images/placeholder.png');
```

Comment:
maybe 302 this so it shares the browser cache?

Reply:
Good plan